README.md

```diff
@@ -2,26 +2,27 @@
 Article templates used within the Guardian’s next-generation iOS and Android applications. This repo also contains documentation that describes the components and layouts used across these templates.
 
 ## Requirements
-* A Mac or Linux PC.
+* A Mac or Linux computer.
 * [brew](http://brew.sh/) as a package manager.
 * [NVM](https://github.com/creationix/nvm).
 * [NodeJS](http://nodejs.org/). Install using nvm: `nvm install v6.9.2`. Remember to add `nvm use v6.9.2` to your preferred shell startup file.
-* [Grunt](http://gruntjs.com/). Install using `npm install -g grunt grunt-cli`.
+* [Yarn] (https://yarnpkg.com). Yarn is a package manager. Install using brew install yarn.
+* [Grunt](http://gruntjs.com/). Install using `yarn global add grunt grunt-cli`.
```

Comment:
@GHaberis Yep! That sounds completely right. It sounds like that would be an easy job. Let me know if it doesn't work for whatever reason.